CU12 and not CU10 in the top paragraph for 2016 (#940)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeServer/new-features/updates.md

::: moniker-end

::: moniker range="exchserver-2016"

> To get the latest version of Exchange 2016, download and install [Cumulative Update 12 for Exchange Server 2016](https://www.microsoft.com/en-us/download/details.aspx?id=57827). Because each CU is a full installation of Exchange that includes updates and changes from all previous CU's, you don't need to install any previous CU's or Exchange 2016 RTM first.

Reviewer: ```suggestion
To get the latest version of Exchange 2016, download and install [Cumulative Update 12 for Exchange Server 2016](https://www.microsoft.com/download/details.aspx?id=57827). Because each CU is a full installation of Exchange that includes updates and changes from all previous CU's, you don't need to install any previous CU's or Exchange 2016 RTM first.
```

> To get the latest version of Exchange 2016, download and install [Cumulative Update 12 for Exchange Server 2016](https://www.microsoft.com/en-us/download/details.aspx?id=57827). Because each CU is a full installation of Exchange that includes updates and changes from all previous CU's, you don't need to install any previous CU's or Exchange 2016 RTM first.

Reviewer: ```suggestion
To get the latest version of Exchange 2016, download and install [Cumulative Update 12 for Exchange Server 2016](https://www.microsoft.com/download/details.aspx?id=57827). Because each CU is a full installation of Exchange that includes updates and changes from all previous CU's, you don't need to install any previous CU's or Exchange 2016 RTM first.
```

::: moniker-end

::: moniker range="exchserver-2016"

> To get the latest version of Exchange 2016, download and install [Cumulative Update 12 for Exchange Server 2016](https://www.microsoft.com/en-us/download/details.aspx?id=57827). Because each CU is a full installation of Exchange that includes updates and changes from all previous CU's, you don't need to install any previous CU's or Exchange 2016 RTM first.

Reviewer 2: ```suggestion
To get the latest version of Exchange 2016, download and install [Cumulative Update 12 for Exchange Server 2016](https://www.microsoft.com/download/details.aspx?id=57827). Because each CU is a full installation of Exchange that includes updates and changes from all previous CUs, you don't need to install any previous CUs or Exchange 2016 RTM first.

```
